internal/operators/mce/ocp_mce_release_matrix.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
I Think this should not be a file, will also removes the code from reading / unmarshal etc..

Reply:
Can you explain "will also removes the code from reading / unmarshal etc..". I think having its own configuration json file will be an advantage as it will need to be updated by some automation.

Reply:
If we want it to make it configurable, I think it should be set as a environment variable like we do with the OS_IMAGES and others.

Reply:
I thought about it, but because it is not in the same package it will require some changes to give access to this variable, which I thought is not worth it as we already do similar things in bump_ocp...

Reply:
@adriengentil WDYT? Does it worth passing around this var for it ?

Reply:
I think it's worth because adding/updating a new channel will just require a config change (in app-sre fo example) and not a complete rollout of the code.

Reply:
Thank you